libs/dh/wholesale/feature-settlement-reports-v2/src/lib/table/dh-settlement-reports-status.component.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, input } from '@angular/core';
+import { SettlementReportStatusType } from '@energinet-datahub/dh/shared/domain/graphql';
+import { WattBadgeComponent } from '@energinet-datahub/watt/badge';
+import { TranslocoDirective } from '@ngneat/transloco';
+
+@Component({
+  standalone: true,
+  selector: 'dh-settlement-reports-status',
+  template: `<ng-container *transloco="let t; read: 'wholesale.settlementReportsV2.reportStatus'">
+    @switch (status()) {
+      @case ('IN_PROGRESS') {
+        <watt-badge type="info">{{ t(status()) }}</watt-badge>
+      }
+      @case ('ERROR') {
+        <watt-badge type="warning">{{ t(status()) }}</watt-badge>
+      }
+      @case ('COMPLETED') {
+        <watt-badge type="success">{{ t(status()) }}</watt-badge>
+      }
```

Comment:
nitpick: according to Figma, the complete status is actually a download link, so this badge is not needed. It's ok for now. It can be replaced in the PR that implements the download functionality.

Reply:
It will be done in the next PR :-)